cfme/base/ssui.py

```diff
@@ -118,7 +118,8 @@ def log_in(self, user, method='click_on_login'):
 @Server.address.external_implementation_for(ViaSSUI)
 def address(self):
     logger.info("USING SSUI ADDRESS")
-    return 'https://{}/self_service/'.format(self.appliance.address)
+    return '{scheme}://{addr}/self_service/'.format(addr=self.appliance.address,
```

Comment:
**Optional**

Consider switching args passed to format to match order of string.

Reply:
done